pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>io.usethesource</groupId>
 	<artifactId>vallang</artifactId>
-	<version>0.15.2-SNAPSHOT</version>
+	<version>1.0.0-SNAPSHOT</version>
```

Comment:
hey, this is quite a jump in versioning. I think we've always said that with semver, 0.x releases the `.x` acts as a major version bump?

Or are we saying: yes, vallang is truly 1.0.0 material now. and we'll be doing full semver?

Reply:
Yes, that's what I'm saying. The API we have now has been very stable anyway, so that's good. More importantly the new fingerprint method is extremely leaky. Literally, every bit we change there will break everything. Hence I think it is important we start observing semver right now.

Reply:
This would be release number 57 by the way. We have 56 `v` tags in the repo now.

Reply:
ok, here we go :)

Reply:
While the compiler project progresses I expect more of the rascal-core run-time to end up in `IValue` and `Type` for the sake of efficiency and for the sake of simplicity (rather a simple dynamic dispatch then constructing visitors all the time). Every time we add such a breaking change, or fix it, we will bump the major version.

Reply:
ok, I think I agree. For the next time, I would have appreciated such a discussion, at least for when we switch from 0.x to 1.0. This might have been something in your head for a while, but for me it came as a suprise, especially added to a PR just before clicking the merge button.

Reply:
ok

Reply:
Since getMatchFingerprint also has to be implemented inside the rascal project, and another method ITree.getConcreteMatchFingerprint too, with similar sensibilities, I believe it is time we also bump the rascal project to 1.0.0. This will allow us to manage the tight integration between the compiler and the run-time with semver. I will repeat this discussion in the next PR.